### PR TITLE
MARCRecord.inc: Add some docs, links to specs, an example

### PR DESCRIPTION
**Pull request by bpfoley**
...so I can begin to understand this.

No functional change.

### CODE REVIEW

File: pinc/MARCRecord.inc [hunk 1]
```diff
@@ -7,11 +7,263 @@ class InvalidMARCRecord extends Exception
...
+        "g" => "Geography",
+        5 => "Historical",
+        6 => "History",
+        7 => "Humor",
+        "i" => "Letter",
+        "l" => "Linguistics",
+        8 => "Math",
+        9 => "Medicine",
+        "m" => "Mixed Form",
+        "v" => 'Music',
```
Comment (srjfoo) [inline pinc/MARCRecord.inc:254]: Should this be `"`Music`"` to match the rest?

File: pinc/MARCRecord.inc [hunk 2]
```diff
@@ -98,24 +354,30 @@ class MARCRecord
     {
         $arthor = "";
```
Comment (srjfoo) [inline pinc/MARCRecord.inc:355]: `$author`?

Comment (cpeel) [reply pinc/MARCRecord.inc:355]: Yeah, this needs to match the other updates.

File: pinc/MARCRecord.inc (hunk 1 above)
Comment (bpfoley) [reply pinc/MARCRecord.inc]: Done!

File: pinc/MARCRecord.inc (hunk 2 above)
Comment (bpfoley) [reply pinc/MARCRecord.inc]: Woops. It most certainly does!

**Review by cpeel — APPROVED** [review]

**Review by srjfoo — APPROVED** [review]

**Review by cpeel — APPROVED** [review]

**Pull request merged**